templates/registration/totp_enable.html

```diff
@@ -103,6 +103,15 @@
                         {{ _('These codes will never be shown again after you enable two-factor authentication.') }}</label>
                     <pre><code><span class="fullwidth">{{ scratch_codes|join('\n') }}</span></code></pre>
                 </div>
+                {% if hardcore %}
+                    <div class="alert alert-danger">
+                        <b>{{ _('Important') }}</b>:
+                        {% trans trimmed site_name=SITE_NAME %}
+                            If you lose both your phone and scratch codes, {{ site_name }} admins will NOT be able to recover your account.
```

Comment:
Maybe change the wording of "phone and scratch codes" to be similar to the one in `two_factor_auth.html`.

Reply:
Fixed.